src/applications/hca/containers/IntroductionPageGated.jsx

```diff
@@ -22,6 +23,19 @@ import {
   shouldShowLoggedOutContent,
 } from '../selectors';
 
+const onVyiBtnClick = (clickEvent, gaEventName) => {
```

Comment:
I _think_ you're doing a lot of extra work here. We have other instances of adding `recordEvent` calls to `<a>` elements by simply calling the function and not needing to worry about calling `preventDefault()` on the click event or using an event callback. For example: https://github.com/department-of-veterans-affairs/vets-website/blob/148b9617b92e86670a1c4094b3c83b95c52745a1/src/applications/personalization/dashboard/containers/DashboardApp.jsx#L56

In the past I've confirmed that things worked by adding a `console.log()` to our `recordEvent` function. But there are more sophisticated ways, such as using a GA Chrome extension. @rianfowler or @ncksllvn  should know what it's called.

Reply:
So we don't need to worry about the GTM-image requests getting canceled, like these?

![gtm-img-canceled-requests](https://user-images.githubusercontent.com/587583/58063129-caf24600-7b31-11e9-9c22-30c56e33f14a.png)

The code I added fixes these canceled requests.  I was reading the [GA docs about how JS stops the instant page-nav starts](https://developers.google.com/analytics/devguides/collection/analyticsjs/sending-hits#knowing_when_the_hit_has_been_sent), but they don't tell you how to check if a "hit" has NOT been sent successfully.

Reply:
interesting... good catch! I think the best bet might be to work with @nedierecel to see if the GA events are getting recorded in her dashboard. If they aren't we likely have multiple places where we are making these calls to GA that get cancelled. Hopefully that isn't the case. It would really suck if we had to do this (very creative!) hack you created because it would potentially slow down, or totally kill, navigation events because each click would have to wait for a response from GA before sending the user to the page they wanted to go to.

Reply:
That is really interesting 🤔 I haven't seen this before, but it does make sense. I assumed that the event was captured quickly enough that it happened before the next page began loading. I'm very interested in if we're possibly missing out on some link clicks in GA because of this. Also curious if this makes any difference between browsers.

Reply:
Page navs are automatically tracked by GA when the URL changes. There's not really a reason to track if someone clicked a button that triggers a page nav.

Reply:
I think sometimes we're interested in what component contained the link they clicked - the header, the footer, etc.

Reply:
That makes sense. I'm curious how the analytics team would use it- I'm vaguely familiar with the path analysis tool in GA and I don't know if you can differentiate between the same link used two times on a page.

Reply:
@nedierecel , I have a couple questions:

1. Do you usually have to wait 'til GA features/fixes are on Staging before you test against your GA dashboard(s)?
2. Do you have a dev.va.gov environment-specific filter/view?

Thanks much!

Reply:
@rianfowler , @ncksllvn , @erikphansen , assuming we do have a GA environment-specific filter for dev.va.gov...

Do we have automatic/scheduled deploys to dev.va.gov, building from master head?

I'd love to test against the actual dashboard eariler.

Reply:
Hello @tlei123,
1. I can test on Dev as well before it goes to staging.
2. There is a Google Analytics Property for dev and staging activity. Here is the direct link to the Dev/Staging property's Real-Time events report. We should be able to see our activity in real-time for this link https://analytics.google.com/analytics/web/?authuser=0#/realtime/rt-event/a50123418w178298118p176884211/

Reply:
I'm actually not sure about analytics on Dev, so I wrote up this [ticket](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/18661) to help get some clarification. If that is working, this would be a good use case for manually building to Dev to test in that environment. Manual builds to Staging generally aren't recommended because it's used so much more frequently. 

But yep, Dev/Staging both deploy automatically from a merge to master. If you'd like, we can merge this, wait for it to go to Staging in about 30 mins, then you can test there and do a follow-up PR based on what you find.

Reply:
@tlei123 master is deployed to dev and staging automatically when PRs are merged into master. 

https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/platform/site-structure/environments/#deployment

Reply:
@tlei123 Per the discussion in https://github.com/department-of-veterans-affairs/vets.gov-team/issues/18661, it looks like you're able to see analytics in Dev via GA Environments. 

If you'd like, you can do a manual deploy to Dev to test the work in this PR. You can do that by -

1. Let folks know on Slack (probably #vetsgov-fe) that you're doing a manual deploy to test in Dev
1. Copy the latest-built commit SHA (Jenkins has to have finished running its whole pipeline on the commit SHA in order to build off of it)
2. Navigate to our Jenkins [vagovdev job](http://jenkins.vetsgov-internal/job/deploys/job/vets-website-vagovdev/)
3. Login (top-right corner)
4. Click "Build with parameters" (left nav)
5. In the ref input box, paste your commit SHA
6. Click "Build"
7. Wait for the build to complete ⌛️

Reply:
Going to add this to the doc site- thanks for putting this together.

Reply:
> @tlei123 Per the discussion in [department-of-veterans-affairs/vets.gov-team#18661](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/18661), it looks like you're able to see analytics in Dev via GA Environments.
> 
> If you'd like, you can do a manual deploy to Dev to test the work in this PR. You can do that by -
> 
> 1. Let folks know on Slack (probably #vetsgov-fe) that you're doing a manual deploy to test in Dev
> 2. Copy the latest-built commit SHA (Jenkins has to have finished running its whole pipeline on the commit SHA in order to build off of it)
> 3. Navigate to our Jenkins [vagovdev job](http://jenkins.vetsgov-internal/job/deploys/job/vets-website-vagovdev/)
> 4. Login (top-right corner)
> 5. Click "Build with parameters" (left nav)
> 6. In the ref input box, paste your commit SHA
> 7. Click "Build"
> 8. Wait for the build to complete ⌛️

Gonna manually deploy this to Dev later today -- I'll coordinate @nedierecel .  Lost internet yesterday.

Reply:
I've removed the nav-delay code from this branch.